pkgs/applications/virtualization/runc/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv, fetchFromGitHub, go-md2man
+, go, pkgconfig, libapparmor, apparmor-parser, libseccomp }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "runc-${rev}";
+  rev = "cc29e3dd"; # this version is used by docker
```

Comment:
So we cannot use the latest release? If the current snapshot is used by docker, that does help, but it's bad if we cannot use the release. (IMHO.)

Package versions should be monotonically increasing. A git hash is not (it jumps all over the place). Please substitute ```${rev}``` with the corresponding commit date (YYYY-MM-DD).